app/views/EditAry/AssessmentRegistryForm/SummaryForm/AddSummaryIssue/AddSummaryIssueModal/index.tsx

```diff
@@ -27,27 +28,41 @@ const CREATE_ASSESSMENT_REGISTRY_SUMMARY_ISSUE = gql`
                 id
                 label
                 subPillar
-                subPillarDisplay
+                subDimension
+                childCount
+                level
+                parent {
+                    id
+                    label
+                    subPillar
+                    subDimension
+                    childCount
+                    level
+                }
             }
         }
     }
 `;
 
 type Props = {
     className?: string;
+    parentId?: string;
     onClose: () => void;
+    refetch: () => void;
 } & ({
     type: 'pillar';
-    subPillar: AssessmentRegistrySummarySubPillarTypeEnum;
+    subPillar?: AssessmentRegistrySummarySubPillarTypeEnum | null;
```

Comment:
I don't think subPillar and subDimesion should null or `?`.

Reply:
This is conditional rendering based upon `type` and `subPillar` or `subDimension` where one must be null or undefined.

Reply:
@puranban , it should be required as the type: pillar | dimension is there to say that subPillar and subDimension are required.

Reply:
```suggestion
    subPillar: AssessmentRegistrySummarySubPillarTypeEnum;
    subDimension: undefined;
```